frontend/talentsearch/src/js/components/search/SearchFilterAdvice.tsx

```diff
@@ -1,61 +1,150 @@
 import React from "react";
 import { useIntl } from "react-intl";
+import { LanguageAbility, Maybe } from "../../api/generated";
 
 const SearchFilterAdvice: React.FC<{
-  classificationFilterCount: number;
   operationalRequirementFilterCount: number;
-}> = ({ classificationFilterCount, operationalRequirementFilterCount }) => {
+  educationSelection: Maybe<boolean>;
+  workingLanguage: Maybe<LanguageAbility>;
+  employmentDuration: Maybe<boolean>;
+  equityFiltersActive: Maybe<boolean>;
+  skillCount: number;
+}> = ({
+  operationalRequirementFilterCount,
+  educationSelection,
+  workingLanguage,
+  employmentDuration,
+  equityFiltersActive,
+  skillCount,
+}) => {
   const intl = useIntl();
   if (
-    classificationFilterCount === 0 &&
-    operationalRequirementFilterCount === 0
+    operationalRequirementFilterCount === 0 &&
+    !educationSelection &&
+    !workingLanguage &&
+    !employmentDuration &&
+    !equityFiltersActive &&
+    skillCount === 0
   ) {
     return null;
   }
 
   const recommendations = [];
-  if (classificationFilterCount > 0) {
+  if (operationalRequirementFilterCount > 0) {
     recommendations.push({
-      key: "classifications",
+      key: "operationalRequirements",
       link: (
         <a
-          href="#classificationsFilter"
+          href="#operationalRequirementFilter"
           data-h2-color="base(dt-primary)"
           data-h2-font-weight="base(700)"
         >
           {intl.formatMessage(
             {
               defaultMessage:
-                "Classification Filters ({classificationFilterCount})",
-              id: "LRysDa",
+                "Conditions of Employment ({operationalRequirementFilterCount})",
+              id: "ky585k",
             },
-            { classificationFilterCount },
+            { operationalRequirementFilterCount },
           )}
         </a>
       ),
     });
   }
-  if (operationalRequirementFilterCount > 0) {
+
+  if (educationSelection) {
     recommendations.push({
```

Comment:
It might be nice to do this in a more functional way, without the array pushes.  I'm thinking something like this:
https://blog.agney.dev/optional-spreading/#spreading-into-an-array